src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+public abstract class Contact {
+
+    public String value;
+    public  boolean isPrivate;
```

Comment:
Maybe consider making private

Reply:
Or maybe it could be protected since the subclasses require access to it. Or follow what David recommended and use the isPrivate() to set the subclasses' isPrivate attribute with it.